dekorator.gemspec

```
@@ -30,7 +30,12 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = Gem::Requirement.new(">= 2.7.0")
 
-  spec.add_runtime_dependency "actionview", ">= 6.0", "< 7.1"
-  spec.add_runtime_dependency "activerecord", ">= 6.0", "< 7.1"
-  spec.add_runtime_dependency "activesupport", ">= 6.0", "< 7.1"
+  spec.add_runtime_dependency "actionview", ">= 6.0", "< 7.2"
+  spec.add_runtime_dependency "activerecord", ">= 6.0", "< 7.2"
+  spec.add_runtime_dependency "activesupport", ">= 6.0", "< 7.2"
+  spec.add_runtime_dependency "railties", ">= 6.0", "< 7.2"
+
+  spec.add_development_dependency "bundler", "~> 2.0"
+  spec.add_development_dependency "rake", ">= 12.3.3"
+  spec.add_development_dependency "rspec", "~> 3.0"
```

```suggestion
  spec.add_runtime_dependency "actionview", ">= 6.0", "< 7.2"
  spec.add_runtime_dependency "activerecord", ">= 6.0", "< 7.2"
  spec.add_runtime_dependency "activesupport", ">= 6.0", "< 7.2"
```

gemspec should be with those lines only

---

Done.